### PR TITLE
Config mail

### PR DESCRIPTION
**Pull request by zelfroster**
Features to customize mail tags and Manage permission for accepting distant mail.

1685779168_Jun03_13:29:28

### CODE REVIEW

File: webui-src/app/config/config_mail.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+      );
+    },
+    view: () =>
+      m('.widget.widget-half.mail', [
+        m('.mail-tags__heading', m('h3', 'Distant Messages')),
+        m('hr'),
+        m('.permission-flag', [
+          m('p', 'Accept encrypted distant messages from: '),
+          m(
+            'select[id=setDistantMessagingPermission]',
```
Comment (rottencandy) [inline webui-src/app/config/config_mail.js:85]: Do we need id here?

Comment (zelfroster) [reply webui-src/app/config/config_mail.js:85]: I thought that I would do it to add some styles, but I didn't need to do anything and forgot to remove it.
Will do it now. 👍🏼

**Comment by zelfroster** [comment]
Done. 👍🏼

**Comment by zelfroster** [comment]
All the commits of this PR are already in the #76 PR. So, Closed it.

**Pull request closed without merging**